question28/css/style.css

```diff
@@ -0,0 +1,231 @@
+    background: #fff;
+    padding: 35px 5%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateY(-50%) translateX(-50%);
+    box-shadow: 3px 6px 10px #999;
+}
+h1 {
+    text-align: center;
+    color: #000;
+    font-size: 30px;
+    margin-bottom: 30px;
+    font-family: 'Didot', sans-serif;
+    letter-spacing: 2px;
+}
+h2 {
+    text-align: center;
+    font-size: 28px;
+    margin-bottom: 25px;
+}
+.modal p {
+    font-size: 15px;
+}
+ol ol {
+    list-style: upper-alpha;
+}
+li {
+    margin-bottom: 20px;
+    list-style: none;
+}
+form li {
+    height: 75px;
+}
+.submit_button {
+    width: 100%;
+    background: #000;
+    font-size: 1rem;
+    line-height: 1.2;
+    font-weight: 300;
+    height: 60px;
+    color: #fff;
+    border-radius: 30px;
+    cursor: pointer;
+    pointer-events: auto;
+}
+.submit_button:disabled {
+    background: #ccc;
+    cursor: not-allowed;
+    pointer-events: none;
+}
+.close_button {
+    width: 50px;
+    height: 50px;
+    background: #000;
+    font-size: 2.5rem;
+    line-height: 1.2;
+    font-weight: 300;
+    color: #fff;
+    border-radius: 30px;
+    position: absolute;
+    top: 0;
+    right: 0;
+    cursor: pointer;
+}
+input {
+    width: 100%;
+    border: 1px solid #80808052;
+    height: 45px;
+    border-radius: 3px;
+}
+.checkbox {
+    margin: 20px 0;
+}
+.checkbox input {
+    height: 25px;
+    width: 25px;
+    vertical-align: middle;
+    margin-right: 8px;
+}
+.modal {
+    max-width: 500px;
+    width: 100%;
+    height: 350px;
+    border-radius: 10px;
+    position: relative;
+    margin: 0 auto;
+    z-index: 2;
+    top: 13%;
+    overflow: hidden;
+}
+.modal_container {
+    width: 100%;
+    height: 100%;
+    background: #fff;
+    overflow: scroll;
+    padding: 30px;
+}
+.modal_content{
+    margin-bottom: 20px;
+}
+.modal.hidden {
+    transform:translate(0,-1300px);
+}
+.mask {
+    background:rgba(0,0,0,0.4);
+    position:fixed;
+    top:0;
+    bottom:0;
+    left:0;
+    right:0;
+    z-index:1;
+}
+.mask.hidden {
+    display: none;
+}
+
+/*///////////////////////////////////
+
+register-done
+
+///////////////////////////////////*/
+
+.register-done {
+    max-width: 800px;
+    padding: 30px;
+    border-radius: 10px;
+    background: #fff;
+    margin: 30px auto;
+    text-align: center;
+    width: 100%;
+    box-shadow: 3px 6px 10px #999;
+}
+.register-done p {
+    color: #353030;
+}
+.register-done_txt {
+    color: #353030;
+    margin-bottom: 2rem;
+}
+
+.register-done_txt p:first-child{
+    font-size: 30px;
+}
+.register-done_link a{
+
+}
+
+
+/*///////////////////////////////////
+
+Validation
+
+///////////////////////////////////*/
+
+input.invalid {
+    border: 2px solid red;
+}
+input.valid {
+    border: 2px solid #94c35d;
+}
+
+.error {
+    color: red;
+    font-size: 12px;
+}
+
+
+/*///////////////////////////////////
+
+login
+
+///////////////////////////////////*/
+.link {
+    margin-top: 30px;
+}
```

Comment:
no need comment
use

.login .link

Reply:
Thank you for your review! Comments were removed and changed to be understood by class name.

fixed commit : https://github.com/csakyo/homework/pull/42/commits/bb08372fb6d872de2e2cc700a06d9e03e0528a62